iOS/Controllers/HomeworkList/HomeworkListCourseSortedViewController.swift

```diff
@@ -59,7 +58,7 @@ final class HomeworkListCourseSortedViewController: UITableViewController {
 
     override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
         let sectionInfo = self.fetchedResultsController.sections![section]
-        guard let view = tableView.dequeueReusableHeaderFooterView(withIdentifier: "HomeworkHeaderView") as? HomeworkHeaderView else {
+        guard let view = tableView.dequeueReusableHeaderFooterView(withIdentifier: R.nib.homeworkHeaderView.name) as? HomeworkHeaderView else {
```

Comment:
`tableView.dequeueReusableHeaderFooterView(withIdentifier: R.nib.homeworkHeaderView)` when previous comment is fixed